ADR/adr-0010-namespace-metadata

```diff
@@ -0,0 +1,39 @@
+# ADR-0010 Namespace Metadata
+
+Date: 2022-December-9
+
+## Status
+
+Accepted
+
+## Approvers
+
+* Ann Marie Fred
+* Gorkem Ercan
+
+## Reviewers
+
+## Context
+
+We need metadata on our namespaces to make Stone Soup easier to operate and maintain. Standardizing namespace metadata will make it easier for us to search our logs and metrics across clusters. It will also allow us to enable logging for outgoing network traffic, one of our security requirements.
+
+## Namespace labels
+
+We will apply the following labels to Stone Soup namespaces, to make them easier to find.
```

Comment:
Needs a definition of what is a "StoneSoup namespace"? For instance does that also include namespaces that we deploy to on BYOC clusters.

Reply:
> 

OK I added a "Naming conventions" section to the index.
I also added more details on which namespaces need the annotations.